src/rh-footer-link.ts

```diff
@@ -1,33 +1,36 @@
-import {html, css, LitElement} from 'lit';
+import { html, css, LitElement } from 'lit';
 
 export class RhFooterLink extends LitElement {
-	static get tag() {
-		return 'rh-footer-link';
-	}
+  static get tag() {
+    return 'rh-footer-link';
+  }
 
-	static get styles() {
-		return css`
-			:host {
-				display: block;
-			}
-			::slotted(*) {
-				color: #fff !important;
-				font-size: 14px;
-				text-decoration: none;
-			}
-		`;
-	}
+  static get styles() {
+    return css`
+      :host {
+        --_color: var(--link-color, #fff);
+        --_font-size: var(--link-font-size, 14px);
+        display: block;
+      }
+      ::slotted(*) {
+        color: var(--_color) !important;
+        font-size: var(--_font-size);
+        text-decoration: none;
+      }
+      ::slotted(a:is(:hover, :focus)) {
+        text-decoration: underline;
+      }
+    `;
+  }
 
-	constructor() {
-		super();
-		this.setAttribute('role', 'listitem');
-	}
+  constructor() {
+    super();
+    this.setAttribute('role', 'listitem');
+  }
 
-	render() {
-		return html`
-			<slot></slot>
-		`;
-	}
+  render() {
+    return html` <slot></slot> `;
+  }
```

Comment:
So curious is upgrading the `aria-role` to `listitem` better than say wrapping the slot with a `li`?

```
render() {
  return html`<li><slot></slot></li>` 
}
```

Then do the same in the `rh-footer-links.ts` instead of `this.setAttribute('role', 'list');` wrapping the whole list in an actual semantic `ul` tag?  

I was taking this exact opposite approach with `rh-secondary-nav`.  In lightdom wanting the user to add aria-roles that way if it fails to upgrade there are at least markers preferably on the nonupgraded wrapper components.  `<rh-secondary-nav-links aria-role="list">` and `<rh-secondary-nav-link aria-role="listitem">` respectively.    Then upgrading to a `<ul><li>` structure in shadow dom when the component does load removing the aria-role tags.  This is a naive question, and I could be lacking an expert understanding of why I wouldn't want to take the approach i was going down.

Reply:
I think that might be a valid alternative.  I would just want to explore if it's valid to have `<li>` to the immediate child of parent `<ul>` since they would be separated by shadow roots.

For posterity. We just had a discussion about this over chat and decided that we would prefer to use native semantic elements in the lightdom in combination with lightdom styles.

```
<rh-footer-links>
  <ul>
    <li>...</li>
  </ul>
</rh-footer-links>
```